assets/scss/components/_header.scss

```diff
@@ -88,10 +88,17 @@ Styleguide Header.Transaction
     text-align: left;
     padding: 0;
   }
+
+  p {
+    font-size: 36px;
```

Comment:
Yeah you can find the core mixins here - https://github.com/hmrc/assets-frontend/blob/master/assets/govuk_frontend_toolkit/stylesheets/_typography.scss#L60
You can pass in a custom line height if you need to but I'd say leave at default unless you really need to.